SCons/Tool/midl.py

```diff
@@ -37,7 +37,7 @@
 import SCons.Scanner.IDL
 import SCons.Util
 
-from .MSCommon import msvc_setup_env_tool
+from SCons.Tool.MSCommon import msvc_setup_env_tool
```

Comment:
why make this change? (and all the similar changes)

Reply:
> The MSCommon module import was changed from a relative import to a top-level/absolute import in the following Microsoft tools: midl, mslib, mslink, mssdk, msvc, msvs.
>
>Moving any of the tools using relative imports to the site tools folder would fail on import (i.e., the relative import paths become invalid when moved).

Should just work moving to scons-site folder.  Was doing some testing and the imports failed.

Reply:
So you'd want moving some part of the msvc logic to site-scons, and not all of it to work?

Reply:
Yes.  If one wanted to copy and customize a microsoft tool like mslib.py locally.  After copying mslib.py to the tools folder in the site-scons folder and running scons, the copied tool import would fail as the relative imports are no longer valid.

The MSCommon module was the only module in the microsoft tools that was using a relative import.  All others were absolute.

This is not something that I'm passionate about.

Reply:
ahh ok.
I think I misread this previously. So this isn't about moving part of a tool to site-tools, but rather a whole tool, which in this case depends on common logic in SCons.Tool

O.k. Then yes these changes make total sense. Good catch.